docs/dapp/tools/band.md

```diff
@@ -10,27 +10,33 @@ platform that aggregates and connects real-world data and APIs to smart
 contracts. You can read more about the specific details of the protocol
 [here](https://docs.bandchain.org).
 
+### Video Tutorial
+
+<p style={{width: '100%'}}>
+<iframe style={{margin: 'auto', display:'block'}} width="560" height="315" src="https://www.youtube.com/embed/cwe6P5MvIfk?si=4QFRA8yJ2PCZB5fq" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</p>
+
 ### Deploy Oracle
 
 1. Follow [this link][demooracle-remix] to Remix. The link contains an encoded example `DemoOracle.sol` contract.
 2. Compile the contract with compiler version `0.6.11`.
 3. Switch to the Deploy tab of Remix.
    1. Select "Injected Web3" in the Environment dropdown in the top left to connect Metamask.
-   2. Make sure that Metamask is connected to the Emerald (Testnet/Mainnet) network. You can read about adding Emerald network to Metamask [here](../../general/manage-tokens/#metamask).
+   2. Make sure that Metamask is connected to an Oasis Paratime (Testnet/Mainnet) network. You can read about adding a network to Metamask [here](../../general/manage-tokens/#metamask).
 
-![Setting up the environment in Remix](../images/emerald/band_demooracle_smartcontract.png)
+![Setting up the environment in Remix](../images/tools/band_demooracle_smartcontract.png)
```

Comment:
Indent since this image is part of the bullet right?

```suggestion
   ![Setting up the environment in Remix](../images/tools/band_demooracle_smartcontract.png)
```

Reply:
Interesting that 2 spaces is okay, but not 3.